contrib/local.mk.example

```diff
@@ -54,6 +54,7 @@
 # CMAKE_EXTRA_FLAGS += -DLIBTERMKEY_USE_STATIC=OFF
 # CMAKE_EXTRA_FLAGS += -DLIBUNIBILIUM_USE_STATIC=OFF
 # CMAKE_EXTRA_FLAGS += -DLIBUV_USE_STATIC=OFF
+# XXX
```

Comment:
Is something else supposed to be here?

Reply:
@jamessan 
Thanks for the review!
Yes, all the other "X_USE_STATIC" usages would get the same treatment as "LIBVTERM_USE_STATIC", and then this would be removed from the example file.

Reply:
=> https://github.com/neovim/neovim/pull/10713

Will use this PR here then for improving libvterm only then.